src/main/webapp/js/maps-page-loader.js

```diff
@@ -7,6 +7,11 @@ const MAP_NIGHT_TYPE_ID = 'night'
  * New modes/map styles can be added in extraMapStyles.json file.
  *
  */
+
+let map;
```

Comment:
I suggest using const instead of let because we're not changing the map object after we initialize it. Also, since we're just building on one single map object, and that map object is just declared and initialized once, I don't think it needs to be removed out of line 23.

Reply:
It is moved out of line 23 so that it becomes a global variable and can be used by other functions I think. If const is used outside, the whole initialisation of `map` needs to be shifted outside

Reply:
Oh really? Coz for me I just passed the map object through a function. But I guess either way works